wp-seo.php

```diff
@@ -1,5 +1,10 @@
 <?php
-/*
+/**
+ * @package    WPSEO
+ * @subpackage Main
+ */
+
+/**
```

Comment:
Ah, wait I though you were refering to top block, not the WP file header.

WP file header part is a mess. I threw in `*` there to make a sniff shut up about it mostly, it needs more tinkering so it both works with sniff and still reliable for WP's parsing of header. I think Juliette just excluded it from sniffing on repo level and I will likely start with that for now.